packages/connection/src/class-plugin.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Plugin connection management class.
```

Comment:
I think it may be interesting to expand the definitions of your classes so it's quickly clear what they are used for. It may save someone some time a few years from now. :)

Reply:
I added more details in the `class` dock blocks. Do you think I should copy the same text to the file blocks, or it would be excessive?
Here's an example of how it looks now: https://github.com/Automattic/jetpack/blob/add/connection-disconnects/packages/connection/src/class-plugin.php